tests/unit/test_charm.py

```diff
@@ -12,9 +12,109 @@ def setUp(self) -> None:
         self.harness.begin()
         self.maxDiff = None
```

Comment:
Similar to the `slurmd` unit tests, I do not believe that you need `self.maxDiff`.

Reply:
Removed